.github/CODEOWNERS

```diff
@@ -1 +1,7 @@
-/.github/workflows/ @JuliaLang/github-actions
+/.github/ @JuliaLang/github-actions 
+/.buildkite/ @JuliaLang/github-actions
+/src/ @JuliaLang/compiler
+/stdlib/LinearAlgebra/ @JuliaLang/linearalgebra
+/stdlib/libblastrampoline_jll/ @staticfloat
+/test/ @JuliaLang/test
```

Comment:
I am not sure this is all that useful since almost any PR adds some tests.

Reply:
My thought was to see how useful this feature is and get it calibrated.